src/editor/port-to-eval-frame.js

```diff
@@ -27,6 +29,61 @@ const approvedKeys = [
   "ctrl+shift+right"
 ];
 
+function validateRequestType(requestType) {
+  if (!["LOAD_FILE", "SAVE_FILE", "DELETE_FILE"].includes(requestType)) {
+    throw Error(`file operation "${requestType}" not defined`);
+  }
+}
+
+function handleFileRequest(message) {
```

Comment:
this might end up going in its own file, so we can better test all the little bits of logic

Reply:
@bcolloran there's no logical place those should go right now. Any suggestions?

Reply:
on second thought maybe just `file-request-actions.js`